packages/react-navigation/lib/create-navigation-container.tsx

```diff
@@ -0,0 +1,40 @@
+import BugsnagPerformance from "@bugsnag/react-native-performance";
+import { NavigationContainer, NavigationContainerProps, NavigationContainerRef, useNavigationContainerRef } from "@react-navigation/native";
+import React, { forwardRef, useRef } from 'react';
+import { NavigationContextProvider } from "./navigation-context";
+
+export const createNavigationContainer = <P extends {}>(Container = NavigationContainer) =>
+  forwardRef<NavigationContainerRef<P>, NavigationContainerProps>(
+    (props, _ref) => {
```

Comment:
Not sure if this `forwardRef` is required? the passed in `_ref` isn't used anywhere

Reply:
We're using `forwardRef` here to access the props of the component provided as a parameter to the function, but I don't think we actually need the ref, as we will generally just be calling this function with the default `NavigationContainer` exported from `@react-navigation/native` so shouldn't have any imperative handling with `refs` before we access it.